### ISSUE TITLE
[Feature Request] Basic aggregations

### ISSUE THREAD

**Issue by DanielPascoe** [dreamRs/esquisse#200]
Some ability to transform the input dataset by way of very basic aggregations might be useful (maybe rendered with a checkbox option to toggle between original dataset and aggregate).

So eg group by one or more columns, then perform one or more aggregate calculations (sum / min / max / mean / mode / 1sd / 2sd etc).

**Comment by pvictor** [comment]
Yes that will be nice, I'll on what I can do, but it will be more likely to be included in package [datamods](https://github.com/dreamRs/datamods). (the complicated thing is to generate code)

**Comment by DanielPascoe** [comment]
Thanks Victor - I guess if implemented via datamods that means aggregations performed on initial import right? Or can one come back to the original dataset and change the operations performed on it? (I tend not to need to transform data when I ues esquisse - I prefer to clean it first for example)
